modules/hostap/src/supp_api.c

```diff
@@ -1499,22 +1520,31 @@ int supplicant_status(const struct device *dev, struct wifi_iface_status *status
 		u8 *_ssid = ssid->ssid;
 		size_t ssid_len = ssid->ssid_len;
 		struct status_resp cli_status;
+#ifndef CONFIG_WIFI_NM_HOSTAPD_AP
 		bool is_ap;
+#endif
 		int proto;
 		int key_mgmt;
+		int sae_pwe;
 
 		if (!ssid) {
 			wpa_printf(MSG_ERROR, "Failed to get current ssid");
 			goto out;
 		}
 
+#ifdef CONFIG_WIFI_NM_HOSTAPD_AP
```

Comment:
`supplicant_status` is only used for NON-hostapd case, why need to `CONFIG_WIFI_NM_HOSTAPD_AP` here?
If `CONFIG_WIFI_NM_HOSTAPD_AP` is undefined, then `sae_pwe` is uninitialized.

Reply:
updated as comment.